python/setup.py

```diff
@@ -35,14 +35,16 @@
 import warnings
 
 
-if "--singlegpu" in sys.argv:
-    from Cython.Build import cythonize
-    from setuptools.command.build_ext import build_ext
-else:
-    try:
-        from Cython.Distutils.build_ext import new_build_ext as build_ext
-    except ImportError:
-        from setuptools.command.build_ext import build_ext
+# if "--singlegpu" in sys.argv:
+from Cython.Build import cythonize
+from setuptools.command.build_ext import build_ext
+# else:
+#     try:
+#         from Cython.Distutils.build_ext import new_build_ext as build_ext
+#     except ImportError:
+#         from setuptools.command.build_ext import build_ext
```

Comment:
I think its better to keep the multigpu normal build enabled, then the PR works as the test for when the libcumlprims cuda 11 build is ready.

Also adding the needed comment: this change is another change that needs to be reverted back before merging the PR

Reply:
Done.